src/django/contricleaner/lib/serializers/row_serializers/row_sector_serializer.py

```diff
@@ -49,7 +49,7 @@ def validate(self, row: dict, current: dict) -> dict:
             current["errors"].extend(sector_errors)
             return current
 
-        splitted_values = split_values(values, ', ')
+        splitted_values = split_values(values, r', |,|\|')
```

Comment:
Let's make possible to configure this values from Django i.e. pass them when we create Django parser...

Reply:
Done